seed/php-sdk/imdb/src/Core/Utils.php

```diff
@@ -39,4 +39,23 @@ public static function castKey(mixed $key, string $keyType): mixed
             default => $key,
         };
     }
+
+    /**
+     * Returns a human-readable representation of the input's type.
+     *
+     * @param mixed $input The input value to determine the type of.
+     * @return string A readable description of the input type.
+     */
+    public static function getReadableType(mixed $input): string
+    {
+        if (is_object($input)) {
+            return 'object(' . get_class($input) . ')';
```

Comment:
I wonder if it's be better to just say the class, e.g. `GetUserResponse` vs. `object(GetUserResponse)`? Not actionable here, just thinking out loud.

Reply:
done!